pom.xml

```diff
@@ -18,34 +18,17 @@
   <properties>
     <java.version>1.8</java.version>
   </properties>
-
+  <!-- TODO (senyer) Fix jar package dependency conflict causing logs to fail to print -->
```

Comment:
???

Reply:
After integrating the distkv client, the console can't print logs.
In my opinion, it may be that the log of distkv conflicts with the log jar of springboot, which makes it can not to print the log. I tried to fix it, but it didn't work.